src/Transport/Connectivity/MessageReceiverCreator.cs

```diff
@@ -22,12 +24,20 @@ public MessageReceiverCreator(IManageMessagingFactoryLifeCycle factories, ReadOn
             var factory = factories.Get(namespaceAlias);
             var receiveMode = settings.Get<ReceiveMode>(WellKnownConfigurationKeys.Connectivity.MessageReceivers.ReceiveMode);
 
-            var receiver = await factory.CreateMessageReceiver(entityPath, receiveMode).ConfigureAwait(false);
+            var prefetchCount = settings.GetOrDefault<int>(WellKnownConfigurationKeys.Connectivity.MessageReceivers.PrefetchCount);
+            var userHasNotProvidedPrefetchCount = !settings.HasExplicitValue(WellKnownConfigurationKeys.Connectivity.MessageReceivers.PrefetchCount);
+            var transportTransactionMode = settings.HasExplicitValue<TransportTransactionMode>() ? settings.Get<TransportTransactionMode>() : settings.SupportedTransactionMode();
```

Comment:
maybe we should create an extension method on settings for this as well, since we now have this check in multiple places?